pkg/plugins/external/init.go

```diff
@@ -40,16 +44,29 @@ func (p *initSubcommand) BindFlags(fs *pflag.FlagSet) {
 }
 
 func (p *initSubcommand) Scaffold(fs machinery.Filesystem) error {
+	configBytes, err := yaml.Marshal(p.config)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Scaffolding with config:", string(configBytes))
```

Comment:
Should we print or you add this one just for debug purposes?

Reply:
Yes it's just for debug purpose, would delete it.

Sorry I should leave a comment to avoid confusion :)